test/e2e/functional/tests/e2e.go

```diff
@@ -727,7 +727,7 @@ var _ = Describe("metallb", func() {
 `))
 	})
 
-	Context("Validate AddressPool Webhook", func() {
+	Context("Validate AddressPool and BGPPeer Webhooks", func() {
```

Comment:
maybe we should separate the AddressPool webhook and the BGPPeer webhook tests to two different contexts? and also maybe adding some basic tests here to test the BGPPeer like we have for the AddressPool webhook?